pkg/examples/readme.md

```diff
@@ -0,0 +1,3 @@
+# rules_pkg - Examples
```

Comment:
Consider naming the file `README.md` to match conventions.

Another alternative for this would be to use the github wiki feature for examples -- might make it easier to contribute.  WDYT?